src/smtlib_reader.cpp

```diff
@@ -24,6 +24,14 @@ using namespace std;
 
 namespace smt {
 
+const unordered_map<string, SortKind> logic_map({ { "LIA", INT },
+                                                  { "NIA", INT },
+                                                  { "LRA", REAL },
+                                                  { "NRA", REAL },
+                                                  { "UF", FUNCTION },
+                                                  { "BV", BV },
+                                                  { "A", ARRAY } });
```

Comment:
What about `QF_AX`?

Reply:
`DT` should probably be in the list, too. Otherwise, parsing a logic string that contains `DT` will fail.

Reply:
Addressed in 1d9005794323a62021ae90160404aa154cc58c53.

Reply:
Missed your comment about `DT`. I might hold off on that actually, because I haven't tested it for parsing datatypes, and tbh I'm not sure that it would work.